app/controllers/hrs_controller.rb

```diff
@@ -0,0 +1,27 @@
+class HrsController < ApplicationController
+    skip_before_action :authenticate_user!, only: [:new, :create]
```

Comment:
Why you skip autho before creating ? Please specify reason

Reply:
if I do not skip before creating, data will not be saved.